Frontend/src/app/pages/product-search-page/product-search-page.component.spec.ts

```diff
@@ -1,10 +1,10 @@
 import {ComponentFixture, TestBed} from '@angular/core/testing';
 
 import {ProductSearchPageComponent} from './product-search-page.component';
-import {HeaderComponent} from "../../components/header/header.component";
-import {FooterComponent} from "../../components/footer/footer.component";
-import {ProductListComponent} from "../../components/product-list/product-list.component";
-import {RouterTestingModule} from "@angular/router/testing";
+import {HeaderComponent} from '../../components/header/header.component';
+import {FooterComponent} from '../../components/footer/footer.component';
+import {ProductListComponent} from '../../components/product-list/product-list.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Mueller-Patrick/Betterzon/pullRequest?prid=7401082)
```suggestion
import {ProductListComponent} from "../../components/product-list/product-list.component";
```